### ISSUE TITLE
Pub social meetup?

### ISSUE THREAD

**Issue by glynhudson** [NorthWalesTech/NorthWalesTech#62]
Is it time for another pub social meetup? Sorry I've not been to the last few events, I've been away or busy. 

Would be great to see everyone and have a chat. Could always combine it with a short talk / film / show and tell to get everyone going.

**Comment by carwyn** [comment]
Yeah, been a bit flat out in December, a social is long overdue. Also trying to pin down speakers for the Spring Tech talks event.

**Comment by carwyn** [comment]
@glynhudson which pub do you think we should go to?

**Comment by glynhudson** [comment]
Mmm not sure, was there a poll on FB? Pontio? Bangor? Parc Meni? Gallt y Glyn (Pizza & pint)?

**Comment by carwyn** [comment]
We had a few of these in the end, this one might have ended up in Gallt y Glyn in Llanberis?
